version/news/NEWS-2024.08.0-cranberry-hibiscus.md

```diff
@@ -93,6 +93,7 @@
 - Fixed a bug where users who attempted login before being provisioned could not log in after being provisioned (rstudio-pro#6564)
 - Fixed bug that prevented Workbench from respecting licenses with an unlimited named user limit when re-activating SCIM provisioned users (rstudio-pro#6561)
 - Fixed bug where user provisioning would not work properly if the homedir column in the database for any user was null (rstudio-pro#6587)
+- Fixed a bug where users could not be found via SCIM provisioning due to incorrectly applied pagination parameters when using SCIM filters (rstudio-pro#6608)
```

Comment:
I would suggest something less technical and closer to the original issue you filed:

```suggestion
- Fixed a pagination issue in the SCIM API that could prevent more than 100 users from being provisioned via Okta (rstudio-pro#6608)
```